src/feeds/FeedReader.ts

```diff
@@ -189,7 +188,9 @@ export class FeedReader {
         log.debug('Loaded feed URLs:', this.observedFeedUrls);
 
         void this.loadSeenEntries().then(() => {
-            return this.pollFeeds();
+            for (let i = 0; i < config.pollConcurrency; i++) {
+                this.pollFeeds(i);
```

Comment:
For consistency with line 406, which I hadn't noticed:
```suggestion
                void this.pollFeeds(i);
```
though its purely a stylistic choice (albeit one that makes it more clear that the returned Promises are meant to be ignored).